### PR TITLE
Misc Fuse.Controls.Native fixes

### PR DESCRIPTION
**Pull request by nekronos**
This PR contains:
- [ ] Changelog
- [ ] Documentation
- [ ] Tests

### CODE REVIEW

File: Source/Fuse.Controls.Native/NativeRenderer.uno [hunk 1]
```diff
@@ -7,7 +7,7 @@ using OpenGL;
 namespace Fuse.Controls.Native
 {
 	extern(Android || iOS)
-	internal interface IViewHandleRenderer : IDisposable
+	public interface IViewHandleRenderer : IDisposable
```
Comment (mortoray) [inline Source/Fuse.Controls.Native/NativeRenderer.uno]: Why does this need to be public?

File: Source/Fuse.Controls.Panels/SingleViewHost.uno [hunk 2]
```diff
@@ -52,6 +52,11 @@ namespace Fuse.Controls
 
 			void NextFrame()
 			{
+				UpdateManager.PerformNextFrame(NextNextFrame);
```
Comment (mortoray) [inline Source/Fuse.Controls.Panels/SingleViewHost.uno]: It'd be cleaner to modify PerformNextFrame to allowing skipping an extra frame -- it already has the ability to wait an arbitrary number of frames internally (`deferFrame`)

I think a comment here would also be helpful as to why waiting two frames is necessary.

File: Source/Fuse.Controls.Native/NativeRenderer.uno (hunk 1 above)
Comment (nekronos) [reply Source/Fuse.Controls.Native/NativeRenderer.uno]: Users cannot implement this interface if it is internal

Comment (mortoray) [reply Source/Fuse.Controls.Native/NativeRenderer.uno]: Are there docs that describe how to use this stuff? (I won't hold up the PR for this reason, just a question)

Comment (nekronos) [reply Source/Fuse.Controls.Native/NativeRenderer.uno]: No docs. This is for advanced users that can read the code.

**Comment by nekronos** [comment]
@mortoray bump

**Comment by mortoray** [comment]
@nekronos Looks good, but your editor changed a lot of whitespace in https://github.com/fusetools/fuselibs-public/pull/437/commits/2e967d4a4e9933660998920c87b903e2c9945a84
Bad setting somewhere?
Shall I accept anyway (I reviewed the actual change, which is fine)

**Comment by mortoray** [comment]
@nekronos Did you still want to get this merged?  It's just the whitespace changes (lots of them) that are an issue.

**Comment by kusma** [comment]
@mortoray: In general, I don't think the white-space changes are a problem on master. I generally object to changes like that on release-branches, due to the needless divergence and risk of merge-conflicts.

Ideally they should come in a separate commit, though.

**Pull request merged**